syntax = "proto3";
package class_schedule;

message Course {
    string name = 1;
    int32 week_period_begin = 2;
    int32 week_period_end = 3;
    int32 class_period_begin = 4;
    int32 class_period_end = 5;
    int32 weekday = 6;
    int32 grade = 7;
    string teacher = 8;
    string place = 9;
}

message CourseList {
    repeated Course course = 1;
}
